extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/ConsumptionDto.java

```diff
@@ -0,0 +1,18 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class ConsumptionDto {
+    final ConsumptionInputDto input;
+    String contractNegotiationId;
+    String transferProcessId;
+    List<String> errors = new ArrayList<>(); //TODO aggregate errors from CN & TP (if not duplicated with dto)
+}
```

Comment:
the entire class is missing swagger annotations with documentation, especially the nullability parameter is important for api clients in languages with clear nullability